forum/communications/utils.py

```diff
@@ -221,3 +221,27 @@ def _create_receivers_namespaces(self) -> list[NamespaceInfo]:
 
     def get_namespace_id(self) -> int:
         return self.namespace.investor_id
+
+
+class ChatNotificationManager(NotificationManager):
+    def __init__(self, namespace_obj: Investor | Startup, room) -> None:
+        super().__init__(namespace_obj)
+        self.room = room
+
+    def _create_receivers_namespaces(self) -> list[NamespaceInfo]:
+        receivers: list[NamespaceInfo] = []
+
+        # TODO: check if receiver allows notification
+        for participant in self.room.participants:
```

Comment:
In this context, we can omit to implement this method, because `ChatNotificationManager` it's an abstract class too. This class is used in pair with `InvestorNotificationManager` or `StartupNotificationManager`(where the method is implemented).